app/assets/stylesheets/application.css

```diff
@@ -46,7 +46,7 @@
   -moz-box-sizing: border-box;
   -webkit-box-sizing: border-box;
   box-sizing: border-box;
-  font-family: "Fira", Helvetica, Arial, sans-serif;
+  font-family: "Fira", Helvetica, Arial, sans-serif !important;
```

Comment:
Does this need to have `!important` on it for sure? It's a good idea to default to not using `!important` expect for in extreme situations where it is absolutely necessary.

Reply:
I was hesitant at 1st, but seems to be the case unless we individually declare the font-family for the elements that need it.
If I remove the `!important`, then these revert back to their old font.
![Screenshot from 2019-06-12 13-39-28](https://user-images.githubusercontent.com/13788498/59317550-86fcf600-8d17-11e9-815e-3b0fb11aeebc.png)
Any idea if there's any other way around it?

Reply:
Ok, cool. That sounds like one of those situations where it might be necessary. I like the idea of having it in one place as opposed to repeated throughout the code.